.vscode/settings.json

```diff
@@ -2,5 +2,9 @@
     "lit-plugin.globalAttributes": ["disabled"],
     "files.associations": {
         "*.svg": "html"
-    }
+    },
+    "editor.formatOnSave": true,
+    "csscomb.formatOnSave": true,
+    "prettier.prettierPath": "./node_modules/prettier",
```

Comment:
I noticed that the auto formatting in VSCode (using the Prettier plugin) sometimes produced different results than `npm run prettier`, which was because VSCode was using a different Prettier version installed elsewhere. This is to make sure we're using a consistent version for formatting.

Reply:
Nice one. I don't have it installed globally, so maybe that's why I didn't run into this.